lib/rules/media-feature-name-value-no-unknown/__tests__/index.js

```diff
@@ -0,0 +1,308 @@
+		{ code: '@media (any-pointer: fine) {}' },
+		{ code: '@media (color-gamut: srgb) {}' },
+		{ code: '@media (color-gamut: p3) {}' },
+		{ code: '@media (color-gamut: rec2020) {}' },
+		{ code: '@media (min-color-index: 15000) {}' },
+		{ code: '@media (max-color-index: 30000) {}' },
+		{ code: '@media (hover: none) {}' },
+		{ code: '@media (hover: hover) {}' },
+		{ code: '@media (overflow-block: none) {}' },
+		{ code: '@media (overflow-block: scroll) {}' },
+		{ code: '@media (overflow-block: paged) {}' },
+		{ code: '@media (overflow-inline: none) {}' },
+		{ code: '@media (overflow-inline: scroll) {}' },
+		{ code: '@media (pointer: none) {}' },
+		{ code: '@media (pointer: coarse) {}' },
+		{ code: '@media (pointer: fine) {}' },
+		{ code: '@media (scan: interlace) {}' },
+		{ code: '@media (scan: progressive) {}' },
+		{ code: '@media (update: none) {}' },
+		{ code: '@media (update: slow) {}' },
+		{ code: '@media (update: fast) {}' },
+		{ code: '@media (dynamic-range: standard) {}' },
+		{ code: '@media (dynamic-range: high) {}' },
+		{ code: '@media (forced-color: none) {}' },
+		{ code: '@media (forced-color: active) {}' },
+		{ code: '@media (horizontal-viewport-segments: 3) {}' },
+		{ code: '@media (environment-blending: opaque) {}' },
+		{ code: '@media (environment-blending: additive) {}' },
+		{ code: '@media (environment-blending: subtractive) {}' },
+		{ code: '@media (inverted-colors: none) {}' },
+		{ code: '@media (inverted-colors: inverted) {}' },
+		{ code: '@media (nav-controls: none) {}' },
+		{ code: '@media (nav-controls: back) {}' },
+		{ code: '@media (prefers-contrast: no-preference) {}' },
+		{ code: '@media (prefers-contrast: more) {}' },
+		{ code: '@media (prefers-contrast: less) {}' },
+		{ code: '@media (prefers-color-scheme: light) {}' },
+		{ code: '@media (prefers-color-scheme: dark) {}' },
+		{ code: '@media (prefers-reduced-data: no-preference) {}' },
+		{ code: '@media (prefers-reduced-data: reduce) {}' },
+		{ code: '@media (prefers-reduced-motion: no-preference) {}' },
+		{ code: '@media (prefers-reduced-motion: reduce) {}' },
+		{ code: '@media (prefers-reduced-transparency: no-preference) {}' },
+		{ code: '@media (prefers-reduced-transparency: reduce) {}' },
+		{ code: '@media (scripting: none) {}' },
+		{ code: '@media (scripting: initial-only) {}' },
+		{ code: '@media (scripting: enabled) {}' },
+		{ code: '@media (vertical-viewport-segments: 3) {}' },
+		{ code: '@media (video-color-gamut: srgb) {}' },
+		{ code: '@media (video-color-gamut: p3) {}' },
+		{ code: '@media (video-color-gamut: rec2020) {}' },
+		{ code: '@media (video-dynamic-range: standard) {}' },
+		{ code: '@media (video-dynamic-range: high) {}' },
+	],
+
+	reject: [
+		{
+			code: '@media screen and (min-width: 1000khz) {}',
+			description: 'Frequency value when only lengths are allowed',
+			message: messages.rejected('min-width', '1000khz'),
+			line: 1,
+			column: 31,
+			endLine: 1,
+			endColumn: 37,
```

Comment:
[ask] I think `endColumn` should be `38`, not `37`... 🤔 Is it expected?
(`"1000khz".length === endColumn - column`)

Reply:
No, not expected :)

Thank you so much for catching this.
`endIndex` needs a `+ 1`